bitcoind-rpc/src/Bitcoin/Core/RPC/Transactions.hs

```diff
@@ -43,7 +52,7 @@ import Servant.API ((:<|>) (..))
 
 import Data.Aeson.Utils (
```

Comment:
Just curious, is this a local module?

Reply:
Yeah, it's an internal module with some aeson related support code.  I use some of the types therein like `HexEncoded` and `Base64Encoded` a lot so it would make sense to break at least those out into an open package.